src/shared/orgListUtil.ts

```diff
@@ -271,6 +271,7 @@ export class OrgListUtil {
         scratchOrgInfo.status = updatedOrgInfo.Status;
         scratchOrgInfo.expirationDate = updatedOrgInfo.ExpirationDate;
         scratchOrgInfo.isExpired = updatedOrgInfo.Status === 'Deleted';
+        scratchOrgInfo.namespace = updatedOrgInfo.Namespace;
```

Comment:
@mshanemc is there a test that can be modified to cover this change?

Reply:
I can NUT-verify the field exists in the output and is null for the scratch orgs it creates.

The problem with namespaces is that when you change hubs, they'd fail (and some hubs might not have a NS so even a query for NS is problematic) so it's more brittle to verify an actual namespace.